pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -136,12 +136,27 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [
     "ibis"
-  ] ++ (map (backend: "ibis.backends.${backend}") backends);
+  ] ++ map (backend: "ibis.backends.${backend}") testBackends;
 
   meta = with lib; {
     description = "Productivity-centric Python Big Data Framework";
     homepage = "https://github.com/ibis-project/ibis";
     license = licenses.asl20;
+    passthru = {
+      extras-require = {
+        clickhouse = [ clickhouse-cityhash clickhouse-driver lz4 ];
+        dask = [ dask pyarrow ];
+        datafusion = [ datafusion ];
+        duckdb = [ duckdb duckdb-engine sqlalchemy ];
+        geospatial = [ geoalchemy2 geopandas shapely ];
+        mysql = [ pymysql sqlalchemy ];
+        pandas = [ ];
```

Comment:
Not sure if we need this when it is empty.

Reply:
It's a feature upstream and used in constructing the test backend list here.

Reply:
I can't find it in https://github.com/ibis-project/ibis/blob/3.0.2/setup.py#L68-L134 . Am I looking at the right repo?

Reply:
Right repo, wrong file 😄: https://github.com/ibis-project/ibis/blob/3.0.2/pyproject.toml#L143